### ISSUE TITLE
LSTM tensorflow to caffe error

### ISSUE THREAD

**Issue by yulinhuyang** [microsoft/MMdnn#841]
Platform (like ubuntu 16.04/win10):ubuntu 16.04

Python version: 3.6

Source framework with version (like Tensorflow 1.4.1 with GPU): tensorflow

Destination framework with version (like CNTK 2.3 with GPU):  caffe

Pre-trained model path (webpath or webdisk path): LSTM

Running scripts: mmconvert -sf tensorflow -iw speaker_detection_model9472.pb --inNodeName input --inputShape 1,25,1 --dstNodeName MobilenetV1/Predictions/Softmax -df caffe -om tf_mobilenet

**Comment by XiaoXYe** [comment]
@yulinhuyang ,thank you for your feedback, MMdnn caffe emitter doesnt have RNN related operations support currently
